build_bundle_image.sh

```diff
@@ -141,6 +146,8 @@ function check_usage {
 		echo "The script reads the following environment variables:"
 		echo ""
 		echo "    LIFERAY_DOCKER_DEVELOPER_MODE (optional): If set to \"true\", all local images will be deleted before building a new one"
+		echo "    LIFERAY_DOCKER_ELASTICSEARCH_NETWORK_ADDRESS (optional): Elasticsearch network address"
+		echo "    LIFERAY_DOCKER_ELASTICSEARCH_PASSWORD (optional): Elasticsearch password"
```

Comment:
@kiwm, can you please give more details here?

Like `Elasticsearch remote server IPv4 address` and `Elasticsearch remote server password`